src/System.Security.Principal.Windows/src/System/Security/Principal/SID.cs

```diff
@@ -50,68 +50,245 @@ internal enum SidNameUse
+        /// <summary>Indicates a SID that matches the Internet built-in user group.</summary>
+        WinBuiltinIUsersSid = 62,
+        /// <summary>Indicates a SID that matches the Internet user group.</summary>
+        WinIUserSid = 63,
+        /// <summary>Indicates a SID that allows a user to use cryptographic operations. It is added to the token of users who are a member of the CryptoOperators built-in group. </summary>
+        WinBuiltinCryptoOperatorsSid = 64,
+        /// <summary>Indicates a SID that matches an untrusted label.</summary>
+        WinUntrustedLabelSid = 65,
+        /// <summary>Indicates a SID that matches an low level of trust label.</summary>
+        WinLowLabelSid = 66,
+        /// <summary>Indicates a SID that matches an medium level of trust label.</summary>
+        WinMediumLabelSid = 67,
+        /// <summary>Indicates a SID that matches a high level of trust label.</summary>
+        WinHighLabelSid = 68,
+        /// <summary>Indicates a SID that matches a system label.</summary>
+        WinSystemLabelSid = 69,
+        /// <summary>Indicates a SID that matches a write restricted code group.</summary>
+        WinWriteRestrictedCodeSid = 70,
+        /// <summary>Indicates a SID that matches a creator and owner rights group.</summary>
+        WinCreatorOwnerRightsSid = 71,
+        /// <summary>Indicates a SID that matches a cacheable principals group.</summary>
+        WinCacheablePrincipalsGroupSid = 72,
+        /// <summary>Indicates a SID that matches a non-cacheable principals group.</summary>
+        WinNonCacheablePrincipalsGroupSid = 73,
+        /// <summary>Indicates a SID that matches an enterprise wide read-only controllers group.</summary>
+        WinEnterpriseReadonlyControllersSid = 74,
+        /// <summary>Indicates a SID that matches an account read-only controllers group.</summary>
+        WinAccountReadonlyControllersSid = 75,
+        /// <summary>Indicates a SID that matches an event log readers group.</summary>
+        WinBuiltinEventLogReadersGroup = 76,
+        /// <summary>Indicates a SID that matches a read-only enterprise domain controller.</summary>
+        WinNewEnterpriseReadonlyControllersSid = 77,
+        /// <summary>Indicates a SID that matches the built-in DCOM certification services access group.</summary>
+        WinBuiltinCertSvcDComAccessGroup = 78,
+        /// <summary>
+        /// Indicates a SID that matches the medium plus integrity label.
+        /// <para><strong>Windows Server�2008�R2, Windows�7, Windows Server�2008, Windows�Vista, Windows Server�2003 and Windows�XP:</strong> This value is not available.</para>
+        /// </summary>
+        WinMediumPlusLabelSid = 79,
+        /// <summary>
+        /// Indicates a SID that matches a local logon group.
+        /// <para><strong>Windows Server�2008�R2, Windows�7, Windows Server�2008, Windows�Vista, Windows Server�2003 and Windows�XP:</strong> This value is not available.</para>
```

Comment:
This comment doesn't correctly apply.  For native it means "you'll get a compile failure if you compile for Windows 7".  For .NET there'd be no such compile failure.

If Windows 7 with .NET Core called `SecurityIdentifier..ctor(WellKnownSidType, SecurityIdentifier)` they'd get an ArgumentException indicating either sidType or domainSid was invalid (quite specific, those being the only two parameters to the ctor), which is fairly close to the current/.NET Framework behavior of ArgumentException that it's not a defined sidType value.

So I think the right answer is to just drop all the comments to the one-sentence summary.